app/src/main/java/com/example/Egida/utils/FirebaseHelper.kt

```diff
@@ -1,9 +1,37 @@
 package com.example.Egida.utils
 
+import com.example.Egida.domain.entity.UserDB
 import com.google.firebase.auth.FirebaseAuth
 import com.google.firebase.auth.FirebaseUser
+import com.google.firebase.database.DatabaseReference
+import com.google.firebase.database.FirebaseDatabase
 
-var AUTH: FirebaseAuth = FirebaseAuth.getInstance()
-var USER: FirebaseUser? = AUTH.currentUser
+lateinit var AUTH: FirebaseAuth
+lateinit var USER: FirebaseUser
+lateinit var REF_DATABASE_ROOT: DatabaseReference
+lateinit var USER_DB: UserDB
```

Comment:
Global access points. And used directly in activity. Bad practice. It should be a repository (constans moved to another class). With an access for only those useCases or viewModels, that really required to have an access.